### ISSUE TITLE
AWS Marketplace images should be pushed to its own registry

### ISSUE THREAD

**Issue by muvaf** [upbound/universal-crossplane#48]
AWS Marketplace requires all container images to be pushed to an ECR repositories that we register in Marketplace so that they can scan for security vulnerabilities and that includes open source images as well, i.e. we'll need to push `crossplane/crossplane` image, too.

In order to do that, we need to have a variant in the build & push pipeline to push to that registry. The tricky part is about how we can use those images in the Helm chart. In https://github.com/upbound/universal-crossplane/pull/22 , I added a [`%%IMAGE_REGISTRY%%`](https://github.com/upbound/universal-crossplane/blob/c31241e/cluster/charts/universal-crossplane/values.yaml.tmpl#L91) variable. We can use that to build an alternative Helm chart with ECR registry and push that to ECR Helm repo and use it in AWS Marketplace installation instructions.

**Comment by muvaf** [comment]
The registry that we push the images (`709825985650.dkr.ecr.us-east-1.amazonaws.com/upbound`) doesn't belong to us so it doesn't show up in our AWS console. I contacted support to ask how to add permissions to a user to push there but haven't heard back.

Today, I just tried giving access to that repo by creating a custom policy that points to that repo, even if it's in another account and the user I assigned it to was able to push an image. So we're unblocked here. I'll add a step to our CI to push the images and update the release guide.
